### PR TITLE
test: Fail when `make check` fails

### PR DESCRIPTION
**Pull request by marusak**
If `make check` failed but `make po` succeeded we would return success.

### CODE REVIEW

**Comment by marusak** [comment]
[debian-testing](https://logs.cockpit-project.org/logs/pull-582-20201029-062208-1e26383d-debian-testing/log.html) is broken :/

**Comment by marusak** [comment]
```
$ sudo systemctl status podman
[sudo] password for admin: 
● podman.service - Podman API Service
     Loaded: loaded (/lib/systemd/system/podman.service; static)
     Active: failed (Result: exit-code) since Fri 2020-10-30 08:17:13 UTC; 15s ago
TriggeredBy: ● podman.socket
       Docs: man:podman-system-service(1)
    Process: 4478 ExecStart=/usr/bin/podman system service (code=exited, status=125)
   Main PID: 4478 (code=exited, status=125)

Oct 30 08:17:13 debian systemd[1]: podman.service: Main process exited, code=exited, status=125/n/a
Oct 30 08:17:13 debian systemd[1]: podman.service: Failed with result 'exit-code'.
Oct 30 08:17:13 debian systemd[1]: Failed to start Podman API Service.
Oct 30 08:17:13 debian systemd[1]: Starting Podman API Service...
Oct 30 08:17:13 debian podman[4478]: time="2020-10-30T08:17:13Z" level=error msg="[graphdriver] prior storage driver overlay failed: remount /var/lib/containers/storage/overlay, flags>
Oct 30 08:17:13 debian podman[4478]: Error: remount /var/lib/containers/storage/overlay, flags: 0x40000: invalid argument
Oct 30 08:17:13 debian systemd[1]: podman.service: Main process exited, code=exited, status=125/n/a
Oct 30 08:17:13 debian systemd[1]: podman.service: Failed with result 'exit-code'.
Oct 30 08:17:13 debian systemd[1]: Failed to start Podman API Service.
```

**Comment by marusak** [comment]
meh, the last commit made things less stable

File: test/run [hunk 1]
```diff
@@ -1,5 +1,4 @@
 #! /bin/bash
 # This is the expected entry point for Cockpit CI; will be called without
 # arguments but with an appropriate $TEST_OS
-make check
-make po/podman.pot
+make check && make po/podman.pot
```
Comment (Gundersanne) [inline test/run:4]: You could also just `set -e` before calling make, it'll fast-fail.

**Comment by marusak** [comment]
omg, fully green! Let me cleanup commits and push again and see, if it persists also in the second run

File: test/run (hunk 1 above)
Comment (marusak) [reply test/run]: of course, thanks for pointing it out! Done

**Comment by marusak** [comment]
soo green :heart_eyes:

**Review by martinpitt — APPROVED** [review]
Looks great, thanks for fixing this!

**Pull request merged**